spec/features/users/sign_in_spec.rb

```diff
@@ -409,9 +409,6 @@
 
         rotate_attribute_encryption_key_with_invalid_queue
 
-        expect { sign_in_user_with_piv(user) }.
```

Comment:
We talked about this change during a PR review meeting and didn't know why we needed this. I'll spend some time with it to make sure we aren't breaking anything.